src/workerd/api/node/tests/crypto_random-test.js

```diff
@@ -413,3 +413,12 @@ export const timingSafeEqualTest = {
     timingSafeEqual(new Uint8Array(1), new Uint8Array(1));
   },
 };
+
+// Ref: https://github.com/cloudflare/workerd/issues/2716
+export const getRandomValuesIllegalInvocation = {
+  async test() {
+    const crypto = await import('node:crypto');
+    const getRandomValues = crypto.getRandomValues;
```

Comment:
Can you also check `const getRandomValues = crypto.webcrypto.getRandomValues`?

I think that this still is not able to be used unbound.

Reply:
Gosh! I was sure I tested that previously but I just tried now and I can confirm that this case does indeed throw.
Thanks!